sonic-gnmi-standalone/debian/opsd-server.substvars

```diff
@@ -0,0 +1,3 @@
+shlibs:Depends=libc6 (>= 2.34)
+misc:Depends=
+misc:Pre-Depends=
```

Comment:
Does this file need to be checked in? I think these are typical generated during build time.